addons/html_builder/static/src/builder/snippets_menu_tabs/invisible_elements_tab.js

```diff
@@ -0,0 +1,104 @@
+import { Component, onWillStart, onWillUpdateProps, useState } from "@odoo/owl";
+import { WithSubEnv } from "../builder_helpers";
+import { getSnippetName } from "../../utils";
+export class InvisibleElementsTab extends Component {
+    static template = "html_builder.InvisibleElementsTab";
+    static components = { WithSubEnv };
+    static props = {
+        invisibleEls: { type: Array },
+        invisibleSelector: { type: String },
+    };
+
+    setup() {
+        this.state = useState({ invisibleEls: null });
+
+        onWillStart(() => this.updateInvisibleElementsPanel(this.props.invisibleEls));
+
+        onWillUpdateProps(async (nextProps) => {
+            this.updateInvisibleElementsPanel(nextProps["invisibleEls"]);
+        });
```

Comment:
a detail but I'd write `nextProps.invisibleEntries`

Reply:
Actually, I wrote it like this to follow a "convention" that we have in website: if a variable stores an html element, its name should end with "El". If the variable stores multiple html elements, it finishes with "Els"